internal/pkg/mapper/relations/load.go

```diff
@@ -23,14 +23,25 @@ func (m *relationsMapper) MapAfterLocalLoad(recipe *model.LocalLoadRecipe) error
 	return nil
 }
 
-func (m *relationsMapper) OnLoad(event model.OnObjectLoadEvent) error {
+func (m *relationsMapper) OnObjectsLoad(event model.OnObjectsLoadEvent) error {
```

Comment:
`OnLoad` event sa volal v `mapper` jednotlivo nad kazdym objektom.
Upravene aby sa `OnObjectsLoad` volal so vsetkymi novymi objektmi naraz, a (`relations`) `mapper` si ich uz iteruje interne.

Takto mame viac moznosti, ... mozeme napr. spustit `goroutine` pre kazdy novy objekt, a cakat kym vsetky dobehnu.
Ked sa to robilo po jednom, tak tato moznost tu nie je.

Reply:
V tomto PR je iba tato zmena, ze sa do `OnObjectsLoad` dostanu cez `event` naraz vsetky nove objekty. Logika sa nemeni, ani sa nic dalsie nepridava.